### PR TITLE
Added properties to signin resource type. Made changes to signin.md a…

### PR DESCRIPTION
**Pull request by danielwood95**
…nd changelog files

### CODE REVIEW

File: api-reference/beta/resources/authenticationRequirementPolicy.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: "authenticationRequirementPolicy resource type"
+description: "Source of authentication requirement, such as Conditional Access, per-user MFA, Identity Protection, and Security Defaults"
+localization_priority: Normal
+author: "danielwood95"
+ms.prod: "microsoft-identity-platform"
```
Comment (davidmu1) [inline api-reference/beta/resources/authenticationRequirementPolicy.md:6]:
```suggestion
ms.prod: "identity-and-access"
```

File: api-reference/beta/resources/authenticationRequirementPolicy.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+doc_type: resourcePageType
+---
+
+# authenticationRequirementPolicy resource type
+
+Namespace: microsoft.graph
+Source of authentication requirement, such as Conditional Access, per-user MFA, Identity Protection, and Security Defaults
```
Comment (davidmu1) [inline api-reference/beta/resources/authenticationRequirementPolicy.md:13]:
```suggestion
Source of authentication requirement, such as Conditional Access, per-user MFA, Identity Protection, and Security Defaults.
```

File: api-reference/beta/resources/authenticationRequirementPolicy.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+
+
+## Properties
+| Property	   | Type	|Description|
+|:---------------|:--------|:----------|
+|requirementProvider|requirementProvider|Enum value representing the source of authentication|
```
Comment (davidmu1) [inline api-reference/beta/resources/authenticationRequirementPolicy.md:20]:
```suggestion
|requirementProvider|requirementProvider| Represents the source of authentication. Possible values: user, request, servicePrincipal, v1ConditionalAccess, multiConditionalAccess, tenantSessionRiskPolicy, accountCompromisePolicies, v1ConditionalAccessDependency, v1ConditionalAccessPolicyIdRequested, mfaRegistrationRequiredByIdentityProtectionPolicy, baselineProtection, mfaRegistrationRequiredByBaselineProtection, mfaRegistrationRequiredByMultiConditionalAccess, enforcedForCspAdmins, securityDefaults, mfaRegistrationRequiredBySecurityDefaults, proofUpCodeRequest, crossTenantOutboundRule, gpsLocationCondition, riskBasedPolicy. |
```

File: api-reference/beta/resources/authenticationRequirementPolicy.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+|detail|String|Source of authentication requirement, such as Conditional Access, per-user MFA, Identity Protection, and Security Defaults|
```
Comment (davidmu1) [inline api-reference/beta/resources/authenticationRequirementPolicy.md:21]: Are the terms after such as actual values? If so, they should be enclosed in code block marks `Conditional Access`.

File: api-reference/beta/resources/signin.md [hunk 5]
```diff
@@ -34,13 +34,17 @@ Provides details about user or application sign-in activity in your directory. Y
...
+|authenticationRequirementPolicies|[authenticationRequirementPolicy](authenticationRequirementPolicy.md) collection|Sources of authentication requirement, such as Conditional Access, per-user MFA, Identity Protection, and Security Defaults.|
 |clientAppUsed|String|The legacy client used for sign-in activity. For example, Browser, Exchange Active Sync,Modern clients, IMAP, MAPI, SMTP, or POP.|
 |conditionalAccessStatus|string| The status of the conditional access policy triggered. Possible values: `success`, `failure`, `notApplied`, or `unknownFutureValue`.|
 |correlationId|String|The identifier that's sent from the client when sign-in is initiated. This is used for troubleshooting the corresponding sign-in activity when calling for support.|
 |createdDateTime|DateTimeOffset|The date and time the sign-in was initiated. The Timestamp type is always in UTC time. For example, midnight UTC on Jan 1, 2014 would look like this: `'2014-01-01T00:00:00Z'`.|
 |deviceDetail|[deviceDetail](devicedetail.md)|The device information from where the sign-in occurred. Includes information such as deviceId, OS, and browser. |
+|flaggedForReview|Boolean|Indicates if user flagged signIns for review.|
```
Comment (davidmu1) [inline api-reference/beta/resources/signin.md:43]:
```suggestion
|flaggedForReview|Boolean|Indicates whether there are user flagged sign-ins for review.|
```

File: api-reference/beta/resources/signin.md [hunk 6]
```diff
@@ -34,13 +34,17 @@ Provides details about user or application sign-in activity in your directory. Y
...
+|homeTenantId|String|The tenantId of the user initiating the signIn. Not applicable in Managed Identity or Service Principal signIns.|
```
Comment (davidmu1) [inline api-reference/beta/resources/signin.md:44]:
```suggestion
|homeTenantId|String|The tenant identifier of the user initiating the sign-in. Not applicable in Managed Identity or Service Principal sign-ins.|
```

File: api-reference/beta/resources/signin.md [hunk 7]
```diff
@@ -56,13 +60,16 @@ Provides details about user or application sign-in activity in your directory. Y
 |riskState|riskState|The risk state of a risky user, sign-in, or a risk event. Possible values: `none`, `confirmedSafe`, `remediated`, `dismissed`, `atRisk`, `confirmedCompromised`, or `unknownFutureValue`.|
 |servicePrincipalId|String|The application identifier used for sign-in. This field is populated when you are signing in using an application.|
 |servicePrincipalName|String|The application name used for sign-in. This field is populated when you are signing in using an application.|
+|signInIdentifier|String|The identification used to signIn - includes UPN, but is also populated when a user logs in in with other identifiers.|
```
Comment (davidmu1) [inline api-reference/beta/resources/signin.md:63]:
```suggestion
|signInIdentifier|String|The identification used to sign in - includes UPN, but is also populated when a user signs in using other identifiers.|
```

File: api-reference/beta/resources/signin.md [hunk 8]
```diff
@@ -56,13 +60,16 @@ Provides details about user or application sign-in activity in your directory. Y
...
+|signInIdentiferType|signInIdentiferType|Provides the type of signInIdentifier. Examples include `userPrincipalName`, `qrCode`, `phoneNumber`, `proxyAddress`.|
```
Comment (davidmu1) [inline api-reference/beta/resources/signin.md:64]:
```suggestion
|signInIdentiferType|signInIdentiferType|Provides the type of sign-in identifier. Examples include `userPrincipalName`, `qrCode`, `phoneNumber`, `proxyAddress`.|
```

File: concepts/changelog.md [hunk 9]
```diff
@@ -67,6 +67,42 @@ For a summary of the value of these API changes, as well as recent tools, compon
 |Addition|beta|Added the **plannerContainerType** enumeration type.|
```
Comment (davidmu1) [inline concepts/changelog.md:67]: We don't need the markdown version of the changelog anymore.

**Comment by jasonjoh** [comment]
Please disregard the earlier message from msftbot. Your pull request was incorrectly flagged.

**Comment by lramosvea** [comment]
@danielwood95 could you please take a look at this PR and address David's comments?

**Pull request closed without merging**